drag_and_drop_v2/templates/html/drag_and_drop.html

```diff
@@ -1,4 +1,4 @@
 {% load i18n %}
-<section class="themed-xblock xblock--drag-and-drop">
+<div class="themed-xblock xblock--drag-and-drop">
     <i class="fa fa-spin fa-spinner initial-load-spinner"></i>{% trans "Loading drag and drop problem." %}
```

Comment:
Thanks, I found two improper usages of `<i>` tag and replaced them with `<span>`s.